packages/core/src/card/InteractableCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { clsx } from "clsx";
+import { ComponentPropsWithoutRef, forwardRef } from "react";
+import { makePrefixer } from "../utils";
+import "./Card.css";
+
+const withBaseName = makePrefixer("saltCard");
+
+export interface InteractableCardProps
+  extends Omit<ComponentPropsWithoutRef<"div">, "onClick"> {
+  /**
+   * If `true`, the card will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * Action called on click.
+   */
+  onClick: React.MouseEventHandler<HTMLDivElement>;
+}
+
+export const InteractableCard = forwardRef<
```

Comment:
The reason why I ask is that if we narrowed the use-case of this component we can be more opinionated in how it works.

Reply:
cc @bhoppers2008

Reply:
Depends how you think about this component... i tend to think of it as a base component which can be used to facilitate various patterns. So it's an interactive version of card, which could be used for navigation, action, selection... just depends what app teams want to do with it.

Reply:
If you view it as a base component, it could work. I think it would make sense to show other use cases possible on the docs. The issue with something being interactable like this is that semantically they are quite different.

If the card was performing an action, then it would be something like a button and therefore would have button semantics, but if it's navigating to a page then it would have link semantics. Selection would get more confusing because it depends if the card is alone or with other cards, then it could have radio or checkbox semantics.

Having a component that could provide all these possible interactions could be confusing, this might not be a problem if the relevant context is given to the component e.g. either behaviour being consistent within an app or the card's content providing context

Reply:
> Selection would get more confusing because it depends if the card is alone or with other cards, then it could have radio or checkbox semantics.

from @bhoppers2008 and I's recent discussion, I think this wouldn't apply - that would be from a SelectableCard component which branched off of InteractableCard (in the future). If we just have the first 2 you mentioned, or similar, interactive actions then it may be easier, but definitely we need to state clearly in the spec what the limit of interactions should be

Reply:
Josh raises a good point /:-| I'm going to contradict myself here. 

I think it's fine that the card doesn't inherit radio/ check/ button semantics, since first and foremost, it's a container and it should be styled as such. We follow the same principle elsewhere as well. Now for the contradiction... we're applying selectable characteristics to the card to create the states atm... so we are already assuming a semantic. 

I believe the styles are correct regardless, since they come from the 'Interact' palette. So i think this gives us two options... we leave it referencing 'Selectable' (maybe a bit hacky?) or we add hover and active tokens to 'Container', which reference 'Interact'.

...or there is another more radical option. We redesign the interactable card 8\ There have been suggestions that the 'Interactable card' should take on a similar aesthetic to what we have on the site. So it would have an accent bar running across the bottom (which would reference accent, but could also be overridden to another colour from the categorical palette). In this case it wouldn't reference selectable at all, and might be [something like this](https://www.figma.com/file/jHTj8hPMa2IL638Mu4kYnH/Untitled?node-id=1%3A25&t=6WUoMDgQUdMid9aS-1).

@pseys @dplsek

Reply:
call scheduled for monday :D